conf/logback.xml

```diff
@@ -15,8 +15,13 @@
         </encoder>
     </appender>
 
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder" />
+    </appender>
```

Comment:
Logs written to stdout are collected by the journal daemon, which in turn is forwarded to Central ELK by [devx-logs](https://github.com/guardian/devx-logs/blob/d55a126f5e0519993b915d3416985b20147ff7d6/ec2/fluentbit/application-logs.conf#L2).